libs/editor/WordPressEditor/build.gradle

```diff
@@ -79,10 +81,13 @@ dependencies {
     api ("com.github.wordpress-mobile.WordPress-Aztec-Android:wordpress-shortcodes:$aztecVersion")
     api ("com.github.wordpress-mobile.WordPress-Aztec-Android:wordpress-comments:$aztecVersion")
 
-    implementation (project(':@wordpress_react-native-bridge')) {
-        exclude group: "com.android.volley"
-        exclude group: 'com.google.code.findbugs'
-    }
+    // The version value here is the same as the one in the repo's root
+    // build.gradle
+    implementation "com.github.wordpress-mobile.gutenberg-mobile:react-native-gutenberg-bridge:20201210-144401"
```

Comment:
Can probably use something like `rootProject.ext.gutenbergMobileVersion` here for the version number.